packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -839,6 +839,13 @@ export default {
                     this.shouldShowSpinner = true;
                 }
             }, this.spinnerTimeout);
+        },
+
+        formFieldBlur (field) {
```

Comment:
minor: not ideal that this is duplication

Reply:
Yes, the method name is the same in x3 files but the source of the data is different and then only a one-liner ... so more code to move to mixin than not ... so defeats the purpose of a mixin, so left as is.